cli/internal/run/run_spec.go

```diff
@@ -74,4 +74,7 @@ type runOpts struct {
 	graphFile     string
 	noDaemon      bool
 	singlePackage bool
+
+	// logPrefix controls whether we should print a prefix in task logs
+	logPrefix string
```

Comment:
This should probably be an enum, similar to TaskOutputMode, but I didn't want to get too fancy yet